hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java

```diff
@@ -300,7 +300,7 @@ private double getMidStripeSplitRatio(long smallerSize, long largerSize, long la
 
   @Override
   public Collection<HStoreFile> getFilesForScan(byte[] startRow, boolean includeStartRow,
-    byte[] stopRow, boolean includeStopRow) {
+    byte[] stopRow, boolean includeStopRow, boolean onlyLatestVersion) {
```

Comment:
The new parameter is not used?

Reply:
StoreFileManager is an interface implemented by DefaultStoreFileManager and StripeStoreFileManager. DefaultStoreFileManager needs this new parameter but StripeStoreFileManager does not currently.  I changed the signature of StoreFileManager#getFilesForScan() to add this new parameter. I am open to suggestions for better handling this.